packages/dds/test-dds-utils/src/test/ddsFuzzHarness.spec.ts

```diff
@@ -1056,5 +1057,49 @@ describe("DDS Fuzz Harness", () => {
 				assert.equal(runResults.stats.failures, 0);
 			});
 		});
+
+		describe("generateTestSeeds", () => {
+			const testCount = 100;
+
+			it("should generate seeds for short stress mode", () => {
+				const seeds = generateTestSeeds(testCount, StressMode.Short);
+				assert.strictEqual(seeds.length, testCount);
+				assert.deepStrictEqual(
+					seeds,
+					Array.from({ length: testCount }, (_, i) => i),
+				);
+			});
+
+			it("should generate seeds for normal stress mode", () => {
+				const seeds = generateTestSeeds(testCount, StressMode.Normal);
+				assert.strictEqual(seeds.length, testCount);
+				assert.deepStrictEqual(
+					seeds,
+					Array.from({ length: testCount }, (_, i) => i),
+				);
+			});
+
+			it("should generate seeds for long stress mode", () => {
+				const seeds = generateTestSeeds(testCount, StressMode.Long);
+				assert.strictEqual(seeds.length, testCount * 2);
+				// Check that seeds are incrementing
+				for (let i = 1; i < seeds.length; i++) {
+					assert.strictEqual(seeds[i], seeds[i - 1] + 1);
+				}
+			});
+
+			it("should generate different seeds for different runs of long stress mode", () => {
+				const seeds1 = generateTestSeeds(testCount, StressMode.Long);
+				const seeds2 = generateTestSeeds(testCount, StressMode.Long);
+				assert.notDeepStrictEqual(seeds1, seeds2);
```

Comment:
```suggestion
				// If this test is ever flaky, consider running multiple trials (as the starting seed is random, sometimes they could be legitimately the same)
				assert.notDeepStrictEqual(seeds1, seeds2);
```

you could do that in this PR if you want, but probably not a huge deal. Just nice to leave a breadcrumb to someone down the line in case this ever does happen.